pmm-app/src/shared/components/Form/FieldAdapters/FieldAdapters.tsx

```diff
@@ -1,6 +1,8 @@
 import React from 'react';
 import { cx } from 'emotion';
-import { Input, TextArea, useTheme } from '@grafana/ui';
+import {
+  Input, Select, TextArea, useTheme
+} from '@grafana/ui';
 import { getStyles } from './FieldAdapters.styles';
 // TODO (nicolalamacchia): use Grafana's components once migration to Grafana v7 is complete
```

Comment:
Isn't that time to act upon this TODO, since we have grafana 7 onboard?

Reply:
One of the reasons of this TODO was to use Field component from Grafana. But we've already seen that we won't use it (in one older PR we argued about it). And since we now have platform-core we will eventually remove all this components, so IMO this can be removed or ignored. But @nicolalamacchia correct me if I'm wrong on this.

Reply:
I'd say it should be _updated_ :) We're gonna use platform-core's components, so FieldAdapters will be removed then. I think we can already use those inputs though, or am I missing something? I see that we're already using them in this PR. In that case I suggest you try them without the FieldAdapter, maybe the style might change a bit, but we can fix it

Reply:
I was almost done with removing InputFieldAdapter from the project but we used a `autocomplete` attribute, which is not allowed in `TextInputField` from platform-core :(. @atymchuk maybe we should add it there too? Or change it so it allows any prop to be passed, or at least any prop of HTML input fields?

Reply:
Yes, sure - any valid HTML input props should be possible to pass. That's a must :)